File: SIMINOV-CONNECT-TEMPLATE/src/siminov/connect/template/events/AuthenticationEventHandler.java
package siminov.connect.template.events;

import siminov.connect.design.authorization.ICredential;
import siminov.connect.events.IAuthenticationEvents;
import siminov.connect.exception.AuthorizationException;
import siminov.orm.resource.Resources;
import android.widget.Toast;

public class AuthenticationEventHandler implements IAuthenticationEvents {

	public void onAuthenticationStart(ICredential credential) throws AuthorizationException {
		
	}

	public void onAuthenticationFinish(ICredential credential) throws AuthorizationException {
		
       Toast.makeText(Resources.getInstance().getApplicationContext(), "Authentication Success", Toast.LENGTH_LONG);
	}

	public void onAuthenticationTerminate(ICredential credential) throws AuthorizationException {
		
	}
}

File: SIMINOV-HYBRID-PHONEGAP-TEMPLATE/assets/www/Liquor/Events/AuthenticationEventHandler.js


function AuthenticationEventHandler() {

	this.onAuthenticationStart = function(credential) {
		alert("onAuthenticationStart");		
	}

	this.onAuthenticationFinish = function(credential) {
		alert("onAuthenticationFinish");
	}

	this.onAuthenticationTerminate = function(credential) {
		alert("onAuthenticationTerminate");
	}
}
